### ISSUE TITLE
Bug when using huber/focal loss  + LDS

### ISSUE THREAD

**Issue by ming197** [YyzHarry/imbalanced-regression#21]
![image](https://user-images.githubusercontent.com/31978733/158639308-7cefaa48-2a9a-4ded-ab0b-81490ac15e6f.png)

In loss.py, "weights" is the last  parameter in function of weighted_huber_loss or weighted_focal

![image](https://user-images.githubusercontent.com/31978733/158639858-3ae3e951-1e7e-4852-8b34-bf3db0c4b6b8.png)

but in train.py, 

loss = globals()[f"weighted_{args.loss}_loss"](outputs, targets, weights)

no atrribute name for weights

maybe it should be 

loss = globals()[f"weighted_{args.loss}_loss"](outputs, targets, weights = weights)

or it would make mistakes when using huber + LDS or focal + LDS

**Comment by YyzHarry** [comment]
Hi - thanks for spotting the issue! The args of these loss functions were intended to keep the order of `outputs`, `targets`, `weights`. We did not carefully check the released code, but it has been updated now (a6fdc45d45c04e6f5c40f43925bc66e580911084).
